CyclopsNuclearReactor/CyNukeReactorSaveData.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace CyclopsNuclearReactor
+{
+    using Common;
+    using Common.EasyMarkup;
+    using SMLHelper.V2.Utility;
+    using System.Collections.Generic;
+    using System.IO;
+
+    internal class CyNukeReactorSaveData : EmPropertyCollectionList<CyNukeRodSaveData>
+    {
+        private readonly string SaveDirectory = Path.Combine(SaveUtils.GetCurrentSaveDataDir(), "CyclopsNuclearReactor");
+
+        private string SaveFile => Path.Combine(SaveDirectory, PreFabId + ".txt");
+
+        private readonly string PreFabId;
+
+        public CyNukeReactorSaveData(string prefabID) : base(prefabID)
```

Comment:
The max number of slots should probably also be something provided to this class to ensure things stay coherent even if users manually mess with the save file..